deps/softwarecontainer-dependencies.sh

```diff
@@ -44,16 +44,27 @@ function install {
 # For softwarecontainer
 install libdbus-1-dev libglibmm-2.4-dev libglibmm-2.4 \
         unzip bridge-utils lcov libjansson-dev libjansson4 \
-        dbus-x11 libcap-dev libtool python3-dev
+        dbus-x11 libcap-dev libtool python3-dev \
+        seccomp libseccomp-dev
 
-apt-get remove --allow-downgrades --allow-remove-essential --allow-change-held-packages -fuy lxcfs lxc2 lxc-dev lxc-common
+apt-get remove --allow-downgrades --allow-remove-essential --allow-change-held-packages -fuy lxcfs lxc2 lxc-dev lxc-common \
+        liblxc-common liblxc-dev lxc1 liblxc1 lxc-templates lxc-tests lxc-utils python3-lxc
 
 # Download and install lxc
 rm -rf lxc
 git clone git://github.com/lxc/lxc -b lxc-3.1.0
 cd lxc
 
 ./autogen.sh
-./configure --prefix=/usr --enable-capabilities --enable-python
+./configure --prefix=/usr --enable-capabilities --enable-python --enable-apparmor --enable-seccomp
```

Comment:
Mention in the commit message that apparmor and seccomp are added to the LXC configuration. The commit message states that unnedded packages are removed, and this is not completely the truth, e.g. instead of installing seccomp and libseccomp-dev, it's built by LXC manually.

Reply:
Added the reason in comment description